tensorflow_io/core/BUILD

```diff
@@ -552,6 +558,7 @@ cc_binary(
         "//tensorflow_io/core:sql_ops",
         "//tensorflow_io/core:text_ops",
         "//tensorflow_io/ignite:ignite_ops",
+        "@libarchive",
```

Comment:
@burgerkingeater Can you remove this line? I think this is not needed.

Reply:
thanks, updated